tests/test_requests.py

```diff
@@ -1145,9 +1145,10 @@ def test_header_keys_are_native(self, httpbin):
     def test_header_validation(self,httpbin):
         """Ensure prepare_headers regex isn't flagging valid header contents."""
         headers_ok = {'foo': 'bar baz qux',
-                      'bar': '1',
+                      'bar': str.encode(u'fbbq'),
```

Comment:
It should be the latter, str.encode() was just what I had in the test previously.